modules/openapi-generator/src/main/resources/kotlin-client/libraries/multiplatform/build.gradle.kts.mustache

```diff
@@ -1,17 +1,17 @@
 import org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget
 
 plugins {
-    kotlin("multiplatform") version "1.5.10" // kotlin_version
-    kotlin("plugin.serialization") version "1.5.10" // kotlin_version
+    kotlin("multiplatform") version "1.5.31" // kotlin_version
+    kotlin("plugin.serialization") version "1.5.31" // kotlin_version
 }
 
 group = "{{groupId}}"
 version = "{{artifactVersion}}"
 
-val kotlin_version = "1.5.10"
-val coroutines_version = "1.5.0"
-val serialization_version = "1.2.1"
-val ktor_version = "1.6.0"
+val kotlin_version = "1.5.31"
```

Comment:
@tkirshboim Why not updating to 1.6.0?

Reply:
@4brunu Jetpack Compose is not yet compatible with Kotlin 1.6.0. Using 1.6.0 would mean that KMM projects that use Compose will have issues using the multiplatform template. Since Compose is a popular UI library on Android, I thought to use 1.5.31, until Compose catches up - something that usually happens quite promptly.

Reply:
You are right 👍 
Could you just update the `serialization_version` to 1.3.0, so that we only use stable versions?